maint-lib/stagelib/envglobals.py

```diff
@@ -73,7 +73,7 @@ def getEnvVar(varname):
 
 
 def exportBaseImageEnvVar():
-    BASE_IMAGE = "{}:{}".format(getEnvVar('BASEOS_REPO'), getEnvVar('BASEOS_TAG'))
+    BASE_IMAGE = "{}:{}".format(getEnvVar('BASEOS_REPO').replace('__', '/'), getEnvVar('BASEOS_TAG'))
```

Comment:
This is a quick solution to this problem. I like it. Can you make sure it gets documented in the staging documentation?

Reply:
Also, this means that `mimic,opensuse__leap,15.0` should be added to the `FLAVORS` spec in the `Makefile`.
If changing the luminous release to be `opensuse__leap`, it also should be changed to `luminous,opensuse__leap,42.3`.

Reply:
The most extensive variable documentation I found is in `envglobals.py`. I added a proper docstring.

Reply:
ok, where do you want that documentation?